src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/TransferTonnage.cshtml

```diff
@@ -55,6 +55,11 @@
         {
             var note = Model.EvidenceNotesDataList.ElementAt(i);
 
+            <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.TransferCategoryValues)">
+                <label for="TransferCategoryValues" class="govuk-visually-hidden">hidden link for navigation to tonnage error</label> <input id="CategoryValues" class="govuk-visually-hidden" />
```

Comment:
Think may just need to update the id from CategoryValues to TransferCategoryValues

Reply:
it is